src/Analyzers/VisualBasic/Analyzers/ParenthesesForMethodInvocations/VisualBasicParenthesesDiagnosticAnalyzerBase.vb

```diff
@@ -0,0 +1,53 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.CodeStyle
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.VisualBasic.CodeStyle
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.ParenthesesForMethodInvocations
+    Friend MustInherit Class VisualBasicParenthesesDiagnosticAnalyzerBase
+        Inherits AbstractBuiltInCodeStyleDiagnosticAnalyzer
+
+        Protected Sub New(diagnosticId As String, title As LocalizableString, message As LocalizableString)
```

Comment:
@CyrusNajmabadi Can the current problem be due to this abstract base class being called with the **same** diagnostic id?
I think need both "Add method paren" and "Remove method paren" having the same id, or am I wrong?

Reply:
I combined both the "Add parentheses" and "Remove parentheses" for methods in only one analyzer, but it's still not working.
Can't even get the breakpoint for InitializeWorker to be hit. Not sure what's wrong.